.github/workflows/test.yaml

```diff
@@ -3,11 +3,9 @@
 #
 name: Tests
 
-# Trigger the workflow's on pushed branches or commits to main/master branch.
 on:
   pull_request:
   push:
-  workflow_dispatch:
```

Comment:
Include this, it provides a button to trigger tests to run via the GitHub ui and can be useful to verify if a PR cause a regression or if tests has stopped working in the main branch for other reasons.

Reply:
Done.